src/pages/index.js

```diff
@@ -33,15 +40,71 @@ export default function Page() {
           {generalSettings?.title} - {generalSettings?.description}
         </title>
       </Head>
-      <Header title="Home Page" />
+      <Header image={mainBanner} />
 
-      <Main className="container">
-        <Posts posts={data?.nodes} readMoreText={'Read More'} id="posts-list" />
-        <LoadMore
-          pageInfo={data.pageInfo}
-          isLoading={isLoading}
-          fetchMore={fetchMore}
-        />
+      <Main className={[styles.home, 'container'].join(' ')}>
+        <section className="hero text-center">
+          <Heading className="heading" level="h1">
```

Comment:
I'm not seeing these styles come through since they are not being imported from `styles`